packages/@lightningjs/ui-components/src/components/Radio/Radio.d.ts

```diff
@@ -32,13 +32,43 @@ export type RadioStyle = {
   strokeWidth: number;
 };
 
-export default class Radio extends Base {
+declare namespace Radio {
+  export interface TemplateSpec extends Base.TemplateSpec {
+    /**
+     * status of the radio button
+     */
+    checked?: boolean;
+  }
+}
+
+declare class Radio<
+  TemplateSpec extends Radio.TemplateSpec = Radio.TemplateSpec,
+  TypeConfig extends lng.Component.TypeConfig = lng.Component.TypeConfig
+> extends Base<TemplateSpec, TypeConfig> {
+  /**
+   * status of the radio button
+   */
   checked?: boolean;
+
   get style(): RadioStyle;
   set style(v: StylePartial<RadioStyle>);
 
   // tags
```

Comment:
This looks great but it seems like we missed adding the 'toggle' method here.

Reply:
Looks like alpha style property is not added in docs file, it would be great if you can add that as well as part of this PR. Thank You!